config/locales/en.yml

```diff
@@ -11,6 +11,7 @@ en:
       one: Follower
       other: Followers
     following: Following
+    self_follow_error: Following your own account is not allowed
```

Comment:
Oh, I missed this was not inserted at the correct place. You can fix that by running `bundle exec i18n-tasks normalize`